### PR TITLE
[R][v1.6.x] Fix blocked pipelines [windows,unix-cpu,unix-gpu] on v1.6.x

### PR DESCRIPTION
**Pull request by ChaiBapchya**
Fixes following issues in the following pipelines

Fix setRefClass not found issue by adding import of library `methods`
Fixes https://github.com/apache/incubator-mxnet/issues/17920
[unix-cpu] - R: CPU
[unix-cpu] - R: MKLDNN-CPU
[unix-gpu] - R: GPU
[website] - R Docs

[website] - Jekyll website
rvm version error [fix by pinning the updated version 2.6.3]

[unix-cpu]  - Tests / Python3: MKLDNN-MKL-CPU 
test_quantize flaky test
Fix : cherrypick the commit from master that fixed this flakiness issue

[unix-gpu] - Static build GPU 14.04 Python
gnu_tls handshake failed [NVIDIA Docker no longer supported on Ubuntu 14.04 hence TLS / certificate issue upon apt-get update]
Fix : switch from https to http

### CODE REVIEW

**Comment by ChaiBapchya** [comment]
@mxnet-bot run ci [unix-cpu]

**Comment by ChaiBapchya** [comment]
Testing this locally, able to reproduce the docs error
```
Loading mxnet
[1] "Loading local: inst/libs/libmxnet.so"
<simpleError in find.package(package, lib.loc, verbose = verbose): there is no package called 'mxnet'
>
[1] "Loading local: src/mxnet.so"
<simpleError in find.package(package, lib.loc, verbose = verbose): there is no package called 'mxnet'
>
Error in loadModule("mxnet", TRUE) : could not find function "loadModule"
Calls: <Anonymous> ... load_code -> <Anonymous> -> run_pkg_hook -> <Anonymous>
Execution halted
```

However, upon checking the local files, both mxnet.so and libmxnet.so exist.
So `dyn.load` isn't able to find package
```
> ls R-package/inst/libs/
libmxnet.so
```
```
> ls R-package/src/
libmxnet.so
```

**Comment by ChaiBapchya** [comment]
Cherrypicking Python2 -> Python3 related PRs I found after doing a git diff between v1.6.x branch and master
https://github.com/apache/incubator-mxnet/compare/v1.6.x...ChaiBapchya:master

**Comment by ChaiBapchya** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by ChaiBapchya** [comment]
So as it turns out, it wasn't needed to cherrypick previous PRs.
Because even without migrating CI from Py2 to Py3 docs test pass
@aaronmarkham @leezu @aaronmarkham 
Plz review/merge. Thanks.

**Comment by ChaiBapchya** [comment]
@lanking520 @zachgk any idea about the Scala failures? http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-cpu/detail/PR-17993/4/pipeline

**Comment by ChaiBapchya** [comment]
@mxnet-bot run ci [unix-cpu]

**Comment by ChaiBapchya** [comment]
Fixes #18005

**Comment by ChaiBapchya** [comment]
It's weird why 1.6 branch is facing rvm version error.
All the previous 5 builds had passed for Jekyll build.

**Comment by ChaiBapchya** [comment]
@mxnet-bot run ci [unix-cpu]

**Comment by zixuanweeei** [comment]
Just a reminder: https://github.com/apache/incubator-mxnet/issues/11747#issuecomment-611913307

**Comment by ChaiBapchya** [comment]
@mxnet-label-bot add [pr-awaiting-review]

**Comment by ChaiBapchya** [comment]
@sandeep-krishnamurthy @zixuanweeei @leezu @aaronmarkham Pl help review.

File: tests/python/quantization/test_quantization.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def check_requantize_with_symbol(shape, min_calib_range=None, max_calib_range=No
                                                                           max_range.asscalar(),
                                                                           min_calib_range=min_calib_range,
                                                                           max_calib_range=max_calib_range)
-        assert_almost_equal(qdata_int8.asnumpy(), qdata_int8_np)
+        assert_almost_equal(qdata_int8.asnumpy(), qdata_int8_np, atol = 1)
```
Comment (leezu) [inline tests/python/quantization/test_quantization.py:181]: Why the change here?

Comment (ChaiBapchya) [reply tests/python/quantization/test_quantization.py:181]: Flaky test test_quantization_mkldnn.test_requantize_int32_to_int8 it failed on 1 of the builds. So I retriggered it. But also added that

Comment (leezu) [reply tests/python/quantization/test_quantization.py:181]: Reference https://github.com/apache/incubator-mxnet/pull/16709
